src/components/large-countdown-clock/SeparatorDots.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { View } from 'react-native';
+import Animated from 'react-native-reanimated';
+import { Flex } from '../layout';
+import styled from '@rainbow-me/styled-components';
+
+const Dots = styled(Animated.createAnimatedComponent(Flex))(
+  ({ minuteEndsWithOne }) => ({
+    alignItems: 'center',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    marginLeft: minuteEndsWithOne ? -1 : 2,
```

Comment:
can we use padding here instead of margin, or nah? Just thinking because the design system doesn't support positive margins on components.

Reply:
This might be a special case but I'll try to do it without a negative margin.